spidermon/templates.py

```diff
@@ -62,16 +76,17 @@ def discover_folder(self, candidate_folder):
             self.add_path(folder)
 
     def reload_env(self):
-        loader = FileSystemLoader(self.paths)
-        self.env = Environment(loader=loader, lstrip_blocks=True, trim_blocks=True)
-        for filter_name, filter in FILTERS.items():
-            self.env.filters[filter_name] = filter
-
-        for global_name, global_value in GLOBALS.items():
-            self.env.globals[global_name] = global_value
+        self.env = get_environment(self.paths)
 
     def get_template(self, name):
-        return self.env.get_template(name)
+        template = None
+        try:
```

Comment:
No need to work here, but we could avoid the exception just by checking if it is an absolute path.
If it is an absolute path, then load with your solution.
Otherwise, keep as before this change.
The current approach, from what I can think, might fail in a scenario (not sure if it is expected, though):
Say that I provided a template name that does not exists, should it raise `TemplateNotFound` or `return None`?
If it should raise the exception, then it will fail in the current approach.
Then, if the idea above holds, it would be nice to add a test case to ensure the expected scenario.